packages/react-components/src/InputNumber.tsx

```diff
@@ -82,7 +82,8 @@ function getSiPowers (si: SiDef | null): [BN, number, number] {
     return [ZERO, 0, 0];
   }
 
-  const basePower = formatBalance.getDefaults().decimals;
+  // const basePower = formatBalance.getDefaults().decimals;
```

Comment:
I'd prefer that we don't have commented code. Unless there's a good reason why we are doing that with a comment that links it to a future task.

Reply:
All good. It just got original Polkadot code be commented, so that it would be easier to track  what the difference is between polkadot apps and cennznet ui